modules/ob-creating-a-s2i-build.adoc

```diff
@@ -0,0 +1,152 @@
+// This module is included in the following assembly:
+//
+// * work-with-builds/using-builds.adoc
+
+:_content-type: PROCEDURE
+[id="ob-creating-a-s2i-build_{context}"]
+= Creating a source-to-image build
+
+You can create a `source-to-image` build and push the created image to a custom Quay repository.
+
+.Prerequisites
+
+* You have installed the {builds-operator} on the {ocp-product-title} cluster.
+* You have created a `ShipwrightBuild` resource.
+* You have installed the `source-to-image` strategy. You can use the following command to verify if the `source-to-image` strategy is installed:
++ 
+[source,terminal]
+----
+$ oc get cbs | grep source-to-image
+----
++
+[NOTE]
+====
+If the `source-to-image` strategy is not installed, see _Installing source-to-image strategy_ in the _Additional resources_ section.
+====
+
+.Procedure
+
+. Create a `Build` resource and apply it to the {ocp-product-title} cluster:
++
+[source,terminal]
+----
+$ oc apply -f - <<EOF     
+apiVersion: shipwright.io/v1beta1
+kind: Build
+metadata:
+  name: s2i-nodejs-build
+spec:
+  source: <1>
+    git:
+      url: https://github.com/shipwright-io/sample-nodejs
+    contextDir: source-build/
+  strategy: <2>
+    name: source-to-image
+    kind: ClusterBuildStrategy
+  paramValues: <3>
+  - name: builder-image
+    value: “quay.io/centos7/nodejs-12-centos7”
```

Comment:
```suggestion
    value: quay.io/centos7/nodejs-12-centos7
```
The `"` is leading to a failure at the later stages, can we please remove these?